frontend/utilities/helpers.tsx

```diff
@@ -783,7 +783,11 @@ export const normalizeEmptyValues = (
   return reduce(
     hostData,
     (result, value, key) => {
-      if ((Number.isFinite(value) && value !== 0) || !isEmpty(value)) {
+      if (
+        (Number.isFinite(value) && value !== 0) ||
+        !isEmpty(value) ||
+        typeof value === "boolean"
```

Comment:
`isEmpty` returns `true` if the value is not an object, array, or string so any boolean value from the API would be considered always empty and replaced with the UI placeholder string "---". 

In its prior [usage](https://github.com/fleetdm/fleet/pull/16879/files#diff-72f7403682d211fc8a84a411fc39c4a33c3eb6a33549a33f1179dd7da6a893ccR766), the boolean value for disk encryption was being passed down directly rather than being run through the normalize function.

Reply:
I was trying to clean up the data being returned by the host and passed down but in turn pushed this unreleased bug.

Reply:
@RachelElysia, I don't think we need to revert. With this change, it should work as expected now :)

Reply:
@RachelElysia, the clean up changes looked great overall! I think they will really help us going forward, especially after we've had a chance to get the types defined for all the summary data fields.